docs/PRODUCTION.md

```diff
@@ -369,3 +369,129 @@ scrape_configs:
 Note: VictoriaMetrics generally performs better than Prometheus.
 
 <!-- TODO: sample dashboards and JSON. -->
+
+## Automatic database backups
+
+You can use [Litestream](https://litestream.io/) for real-time database backups.
+
+### Installation
+
+1. Install Litestream.
+
+    ```bash
+    wget -qO- https://github.com/benbjohnson/litestream/releases/download/v0.3.9/litestream-v0.3.9-linux-amd64-static.tar.gz | sudo tar xzvf - --no-same-owner -C /usr/local/bin litestream
+    ```
+
+2. Create a template systemd service and configuration directory.
+
+    ```bash
+    # note: we restrict the permissions since the configs usually contain credentials
+    sudo mkdir --mode=0700 /etc/litestream
+    sudo nano /etc/systemd/system/litestream@.service
+    ```
+
+    ```ini
+    [Unit]
+    Description=Litestream (%I)
+
+    [Service]
+    User=root
+    Group=root
+
+    Restart=always
+    ExecStart=/usr/local/bin/litestream replicate -config /etc/litestream/%I.yml
+
+    ProtectClock=yes
+    ProtectKernelTunables=yes
+    ProtectKernelModules=yes
+    ProtectKernelLogs=yes
+    ProtectControlGroups=yes
+    PrivateTmp=yes
+    PrivateDevices=yes
+    PrivateMounts=yes
+
+    [Install]
+    WantedBy=multi-user.target
+    ```
+
+3. Create the configuration for Atlas.
+
+    ```bash
+    sudo nano /etc/litestream/atlas.yml
+    ```
+
+    ```yml
+    dbs:
+    - path: /var/lib/atlas/accounts.db
+      replicas:
+      - url: sftp://USER:PASSWORD@SERVER:PORT/litestream/accounts
+        retention: 336h
+        snapshot-interval: 24h
+        sync-interval: 300s
+    - path: /var/lib/atlas/pdata.db
+      replicas:
+      - url: sftp://USER:PASSWORD@SERVER:PORT/litestream/pdata
+        retention: 336h
+        snapshot-interval: 24h
+        sync-interval: 300s
+    ```
+
+    Note: Update the database paths as required. You can also use other kinds of replicas or change the times as described [here](https://litestream.io/reference/config/).
+
+    Note: You will need enough free disk space to store `snapshot-interval` worth of database changes locally.
```

Comment:
Do we have a rough estimate how much that is in our current case?

I know that highly depends on user count and their interactions but a rough ballpark (KB, MB, GB) would be nice to have.

Don't bother if it takes more than 5 minutes to figure out. ^^

Reply:
Well, based on our metrics, atlas writes an average of 77 gigabytes per month, which is about 2.6 gigabytes per day, subtract 1.4 gigabytes of logs on average, which leaves about 1.1 gigabytes of database writes per day, which is about .046 gigabytes per hour, multiplied by 336 hours, equals about 15.5 gigabytes of WAL, plus .6 gigabytes for the entire database times 336/24 hours, equals about **24 gigabytes of backup storage** and .046 times 24 hours, equals about **1.2 gigabytes of local WAL storage** (but usually MUCH lower if litestream is working properly) at worst. This will **increase by 0-1% per month** based on our current growth rate.